taskmanager/features/task.feature

```diff
@@ -72,5 +72,5 @@ Feature: Task management
 
   Scenario: Change task status from CLOSED to TODO
     Given there is a task named "Add switch language button" with "CLOSED" status
-    When user named "Sarah" changes task named "Add switch language button" status to "TODO"
-    Then task named "Add switch language button" should have status "CLOSED"
+    When task "UnexpectedStatusChangeException" is thrown during user named "Sarah" is changing task named "Add switch language button" status change to "TODO"
```

Comment:
e.g.
```gherkin
When I try to change status from CLOSED to TODO
Then task still should have closed status
And I should see notice that this action is disallowed
```

Reply:
@tarnawski @partikus thanks for your comments. I started with similar scenarios but I have problem with implementation of `When` and `And` steps.
I assume it should be something like
```
public function WHEN() {
        try {
            $task->setStatus($status, $user);
        } catch (UnexpectedStatusChangeException $exception) {
            // notify user - how? echo?
        }
}

public function AND() {
    // how to check that user was notified?
}
```